python/lsst/summit/utils/imageExaminer.py

```diff
@@ -451,7 +456,7 @@ def plotRadialAverage(self, ax: matplotlib.axes.Axes | None = None) -> None:
 
         ax.plot(distances, values, "x", label="Radial average")
         if not fitFailed:
-            fitline = gauss(distances, *pars)
+            fitline = gauss(np.array(distances), *pars)
```

Comment:
`distances` is a list of floats, and that obviously works, because the code runs fine, so I think in cases like that it's better to add to the annotations rather than doing an unnecessary cast, so I'd just add that lists are OK for the `gauss` function.

Reply:
list[float] - float is not accepted... And I dont know why its working with a list of float, I can remove the conversion and add a type: ignore, sincerelly is the only idea I have